problems/loops.js

```diff
@@ -143,16 +228,33 @@ function fizzBuzzPart1() {}
  * @returns {Array}
 
  */
-function fizzBuzzPart2() {}
-
+const fizzBuzzPart2 = () => {
+  let newArray = [];
+  for (let i = 1; i <= 100; i++) {
+    if (i % 3 === 0) {
+      newArray.push("Fizz");
+    } else if (i % 5 === 0) {
+      newArray.push("Buzz");
+    } else if (i % 3 === 0 && i % 5 === 0) {
```

Comment:
Revisit the ordering of these. Remember once one if is satisfied that ends it won't check the other conidtions.